appgateway_create.yml

```diff
@@ -95,18 +82,18 @@
 
     - name: Create a public IP address
       azure_rm_publicipaddress:
-        resource_group: "{{ resource_group }}"
+        resource_group: "{{ resource_group }}" 
         allocation_method: Dynamic
         name: appgwpublicip
         domain_name_label: "{{ resource_group }}"
       register: pip_output
 
-    - name: Dump Public IP output
+    - name: Dump Public IP output 
       debug:
         var: pip_output
 
     - name: Create instance of Application Gateway
-      azure_rm_appgw:
+      azure_rm_appgateway:
```

Comment:
In 2.7, the name is azure_rm_appgateway. But in roles, the nae is azure_rm_appgw. We should align them.